### ISSUE TITLE
obs_sim.storage_max_p_absorb is None in sim2real env with Lightsim

### ISSUE THREAD

**Issue by marota** [rte-france/Grid2Op#518]
## Environment
 - Grid2op version: `1.9.4`
 - System: `ubuntu 20.04.3 LTS, ...`

## Bug description
obs.storage_max_p_absorb attribute is None for a simulated obs when using a sim2real environment with LightSIm2grid backend

### Command line

```bash
# command line used if any 
```

### Code snippet

```python
from grid2op import make
from lightsim2grid import LightSimBackend

BACKEND = LightSimBackend
env=make("input_data_local_sim2real",backend=BACKEND()) #an env from sim2real track
obs,reward,done,info=env.step(env.action_space({}))
obs_simu,reward,done,info=obs.simulate(env.action_space({}),time_step=1)
assert(obs_simu.storage_max_p_absorb is not None)
```
I tried to reproduce also with env "l2rpn_case14_sandbox_diff_grid" but I don't get this behavior probably because there are no storage in the first place on that grid

## Current output

```
obs_simu.storage_max_p_absorb is None
```

## Expected output

```
obs_simu.storage_max_p_absorb should be the same as obs.storage_max_p_absorb
```

**Comment by marota** [comment]
obs.storage_Emax is also impacted for simulation obs

**Comment by BDonnot** [comment]
fixed in version 1.9.5 (released a few minutes ago)
